### PR TITLE
changed to actual expected values after .to_i

### PR DESCRIPTION
**Pull request by Montana-Pfeifer**
changed to eq to be actual expected values.

### CODE REVIEW

File: spec/stat_tracker_spec.rb [hunk 1]
```diff
@@ -85,13 +85,13 @@
       expect(@stat_tracker1.get_scores(6, :away)).to eq([2, 3, 3, 4])
       expect(@stat_tracker1.get_scores(6, :total)).to eq([3, 3, 3, 2, 1, 2, 3, 3, 4])
       expect(@stat_tracker1.get_scores(6, :blahblah)).to eq([3, 3, 3, 2, 1, 2, 3, 3, 4])
-      expect(@stat_tracker1.highest_total_score).to eq("41")
+      expect(@stat_tracker1.highest_total_score).to eq(5)
     end
   end
  
   describe '#lowest_total_score' do
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: Is this a duplicate of the test on line 73?

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -116,13 +116,13 @@
...
     it 'can find the lowest scoring visiting team' do
       expect(@stat_tracker2.lowest_scoring_visitor).to eq('New York Red Bulls')
-      expect(@stat_tracker1.highest_total_score).to eq("41")
+      expect(@stat_tracker1.highest_total_score).to eq(5)
     end
   end
 
   describe '#lowest_total_score' do
     it 'returns the lowest sum of the winning and losing teams’ scores' do
-      expect(@stat_tracker1.lowest_total_score).to eq("03")
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb:125]: Is this a duplicate of the test on line 73?

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -116,13 +116,13 @@
   describe '#lowest_scoring_visitor' do
     it 'can find the lowest scoring visiting team' do
       expect(@stat_tracker2.lowest_scoring_visitor).to eq('New York Red Bulls')
-      expect(@stat_tracker1.highest_total_score).to eq("41")
+      expect(@stat_tracker1.highest_total_score).to eq(5)
```
Comment (wally-yawn) [inline spec/stat_tracker_spec.rb]: I don't think this line belongs to this test

**Review by Crosswolfv1 — APPROVED** [review]
Removed the duplicate tests cause by merging and fixed existing tests to search for proper values

**Pull request merged**